open-amt-cloud-toolkit/docs
# docs(api): test swagger plugin

## PR Checklist


- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

- Test out the swagger plugin for integrating into our docs site.

## Anything the reviewer should know when reviewing this PR?
To test locally:
- Install plugin with `pip install mkdocs-render-swagger-plugin`, [see plugin repo here](https://github.com/bharel/mkdocs-render-swagger-plugin).
- Start server with `mkdocs serve`
